src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -970,19 +971,19 @@ Parameter Plugin::GetMetric(const std::string& name, const std::map<std::string,
         std::vector<ov::PropertyName> cachingProperties;
         cachingProperties.push_back(ov::PropertyName(uarch_version.name(), PropertyMutability::RO));
         cachingProperties.push_back(ov::PropertyName(execution_units_count.name(), PropertyMutability::RO));
-        cachingProperties.push_back(ov::PropertyName("GPU_DRIVER_VERSION", PropertyMutability::RO));
+        cachingProperties.push_back(ov::PropertyName(driver_version.name(), PropertyMutability::RO));
```

Comment:
what about storing device name and device id as well, to be sure?

Reply:
Even though the device id is different, if uarch and the number of eu are the same, wouldn't it be possible to share the same cache blob?

Reply:
Is it required from usage perspective? If not, I would recommend not to do so.
Actually, even if the device names are the same, it does not mean that the devices are actually "same". It may be from different stepping and different WA might have been applied from driver.(not sure about this) So the safe approach would be not to allow if we are not really sure.

Reply:
I added `device_id` in the `caching_properties`. Thank you.